promql/promqltest/testdata/functions.test

```diff
@@ -6,18 +6,15 @@ load 5m
 
 # Tests for resets().
 eval instant at 50m resets(http_requests[5m])
-	{path="/foo"} 0
-	{path="/bar"} 0
-	{path="/biz"} 0
```

Comment:
It's good to test that the 5m range yields an empty result, but let's also add a new test with 10m range, to show that it has the same result as the previous 5m test.